cg/meta/workflow/balsamic.py

```diff
@@ -585,7 +593,8 @@ def config_case(
                 "--swegen-sv": arguments.get("swegen_sv"),
                 "--tumor-sample-name": arguments.get("tumor_sample_name"),
                 "--umi-trim-length": arguments.get("umi_trim_length"),
-            }
+            },
+            exclude_true = True
```

Comment:
haha no! I thought I would need it because of how this function works:

```
def build_command_from_dict(options: dict, exclude_true: bool = False) -> list[str]:
    """Returns a command list of strings given a dictionary with arguments."""
    formatted_options: list = []
    for option_name, option_value in options.items():
        if option_value:
            if exclude_true and option_value is True:
                formatted_options.append(str(option_name))
            elif option_value:
                formatted_options.append(str(option_name))
                formatted_options.append(str(option_value))
    return formatted_options
```
If I understand it correctly it only adds the name of the option "--exome" if the value is set to True, and exclude_true is set to True. So it removes True, which I think is necessary to make the --exome flag work. Probably I misunderstood something :D

Reply:
No, it's me not being aware of that option, sorry 🤦 

Nice and clean implementation ⭐